### DISCUSSION TITLE
ScreenShot tool not Found in "Customize Tool Bar" menu

### DISCUSSION THREAD

**Discussion by raindady** [Floorp-Projects/Floorp#712] [Feature Request]
ScreenShot tool not Found in "Customize Tool Bar" menu
Firefox menu Screenshot attached
![Screenshot tool](https://github.com/Floorp-Projects/Floorp/assets/156963295/d89ed71f-1a52-4c32-8dd5-3c7fc9702e4b)

**Comment by Ryosuke-Asano** [comment]
See status bar

**Comment by Telmesomething** [comment]
It's down at the bottom, left hand corner. You can grab it with the mouse and add it to the Customize menu from there, or drag it onto the toolbar.
